dlrover/python/tests/test_elastic_training_agent.py

```diff
@@ -432,6 +436,116 @@ def test_sync_node_port_with_env(self):
             str(65000),
         )
 
+    def test_stop_workers_ascend(self, cmdline=None):
+        # test Ascend NPU
+        config = self.config
+        spec = self.spec
+
+        print(cmdline)
+        self.config.accelerator = Accelerators.ASCEND_NPU
+        self.spec.max_restarts = 0
+        if cmdline is None:
+            self.spec.entrypoint = "sleep"
+            self.spec.args = tuple(["180"])
+        else:
+            self.spec.entrypoint = cmdline[0]
+            self.spec.args = tuple(cmdline[1:])
+
+        self.config.network_check = False
+        agent = ElasticTrainingAgent(
+            node_rank=0,
+            config=self.config,
+            entrypoint=self.spec.entrypoint,
+            spec=self.spec,
+            start_method=self.config.start_method,
+            log_dir=self.config.log_dir,
+        )
+
+        def stop_task(agent):
+            print("entering stop_task")
+            time.sleep(90)
```

Comment:
Why 90 seconds sleep here? The unit test's efficiency also needs to be considered.